www/docs/server/procedures.md

```diff
@@ -157,6 +157,27 @@ export const appRouter = router({
 export type AppRouter = typeof appRouter;
 ```
 
+### With [@effect/schema](https://github.com/Effect-TS/schema)
+
+Wrap your schema in `S.parse` to make it compatible with tRPC.
```

Comment:
It's not really 'wrapping' per-se, the model `schema` uses means that you create a schema, which is like a runtime AST for defining transformation. You then supply these schemas to parser constructors (`parse`, `parsePromise`, `decode`, `encode`, `is`, `assert`, or others). This returns a function that _does-the-thing_.

Basically I'm being pretentious about the terminology "wrap", I'd use 
> "`@effect/schema` exposes `S.parse` to parse a schema from `unknown`, this would make it compatible with tRPC.",

 and then the example